packages/foundry/package.json

```diff
@@ -7,8 +7,8 @@
     "fork": "anvil --fork-url ${0:-mainnet} --config-out localhost.json",
     "compile": "forge compile",
     "generate": "node script/generateAccount.js",
-    "deploy": "forge build --build-info --build-info-path out/build-info/ && forge script script/Deploy.s.sol --rpc-url ${0:-default_network} --broadcast && node script/generateTsAbis.js",
-    "deploy:verify": "forge build --build-info --build-info-path out/build-info/ && forge script script/Deploy.s.sol --rpc-url ${0:-default_network} --broadcast --verify && node script/generateTsAbis.js",
+    "deploy": "forge build --build-info --build-info-path out/build-info/ && forge script script/Deploy.s.sol --rpc-url ${1:-default_network} --broadcast --legacy && node script/generateTsAbis.js",
```

Comment:
Why is the legacy flag is needed?

Reply:
It is about [EIP 1559](https://eips.ethereum.org/EIPS/eip-1559), by default foundry will try to use EIP 1559 transactions, but some RPCs dont support it so transactions would fail. Has happened to me in some mumbai and custom network RPCs. For now it is more stable to have `--legacy`.